starters/nuxt/package.json

```diff
@@ -12,6 +12,7 @@
     "lintfix": "npm run lint:js -- --fix"
   },
   "dependencies": {
+    "@nacelle/shopify-cart": "^0.2.1",
```

Comment:
1. `@nacelle/shopify-checkout` should be removed.
2. We need the `package-lock.json` updated. Try `npm i --package-lock.json` and bootstrapping again to make sure everything is in sync.